x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/details_page/components/package_policies/package_policies_table.tsx

```diff
@@ -224,7 +224,7 @@ export const PackagePoliciesTable: React.FunctionComponent<Props> = ({
           }
         ),
         render: (namespace: InMemoryPackagePolicy['namespace']) => {
-          return namespace ? (
+          return (namespace && namespace != agentPolicy.namespace) ? (
```

Comment:
I think the fix is not correct here, we don't want to display the tooltip if the integration policy has its own, even if it is the same as the agent policy's.
The issue happens on the backend, when auto creating a system integration with an agent policy, I think the "default" namespace is set on the integration policy, so it should be removed from there for consistency.

Reply:
I think this line should be removed to fix the issue: https://github.com/elastic/kibana/blob/300b8ee3683663c9da530ecb41e28731b194b56e/x-pack/plugins/fleet/server/services/agent_policy_create.ts#L74